### PR TITLE
Update dlp-on-premises-scanner-get-started.md

### PR DESCRIPTION
**Pull request by braalf**
Adding license information on who needs a license to use the scanner.

### CODE REVIEW

**Comment by braalf** [comment]
@chrfox

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/dlp-on-premises-scanner-get-started.md [hunk 1]
```diff
@@ -39,6 +39,9 @@ Before you get started with DLP on-premises scanner, you should confirm your [Mi
 
 For full licensing details see: [Microsoft 365 licensing guidance for security & compliance](/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance/microsoft-365-security-compliance-licensing-guidance)
 
+> [!IMPORTANT]
+> All users that contribute to the scanned location either by adding files or consuming files need to have a license.  Not just the scanner user.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-on-premises-scanner-get-started.md]:
```suggestion
> All users who contribute to the scanned location either by adding files or consuming files need to have a license, not just the scanner user.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@braalf Please implement the suggested changes. Thanks

**Comment by braalf** [comment]
@yogkumgit updated with suggested change.  Thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Pull request merged**